examples/hydration-smiles/freesolv.yaml

```diff
@@ -0,0 +1,49 @@
+---
+options:
+  minimize: yes
+  verbose: yes
+  output_dir: .
+  number_of_iterations: 5000
+  temperature: 300*kelvin
+  pressure: 1*atmosphere
+  softcore_beta: 0.0
+
+molecules:
+  molecule:
+    filepath: freesolv-mini.smiles
+    openeye:
+      quacpac: am1-bcc
+    antechamber:
+      charge_method: null
+    select: all
+
+solvents:
+  RF:
+    nonbonded_method: PME
+    nonbonded_cutoff: 9*angstroms
+    clearance: 9*angstroms
+  vacuum:
+    nonbonded_method: NoCutoff
+
+systems:
+  hydration-system:
+    solute: molecule
+    solvent1: RF
+    solvent2: vacuum
+    leap:
+      parameters: [leaprc.ff14SB, leaprc.gaff]
+
+protocols:
+  hydration-protocol:
+    solvent1:
+      alchemical_path:
+        lambda_electrostatics: [1.00, 0.75, 0.50, 0.25, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00]
+        lambda_sterics:        [1.00, 1.00, 1.00, 1.00, 1.00, 0.95, 0.90, 0.85, 0.80, 0.75, 0.70, 0.65, 0.60, 0.50, 0.40, 0.30, 0.20, 0.10, 0.00]
+    solvent2:
+      alchemical_path:
+        lambda_electrostatics: [1.00, 0.75, 0.50, 0.25, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00, 0.00]
```

Comment:
I see the FreeSolv paper didn't, so it's probably best to refer to [this earlier paper](http://pubs.acs.org/doi/abs/10.1021/ct800409d) for protocol.